src/main/java/org/opentripplanner/routing/algorithm/raptor/router/AdditionalSearchDays.java

```diff
@@ -0,0 +1,76 @@
+package org.opentripplanner.routing.algorithm.raptor.router;
+
+import java.time.Duration;
+import java.time.Period;
+import java.time.ZonedDateTime;
+import java.util.Optional;
+
+public class AdditionalSearchDays {
```

Comment:
All public classes require a short Javadoc explaining what the class is for

Reply:
Done.